web/client/utils/cesium/MathUtils.js

```diff
@@ -7,6 +7,7 @@
 */
 
 import * as Cesium from 'cesium';
+import earcut from 'earcut';
```

Comment:
Is this lib included? I can not see it package.json

Reply:
@offtherailz I think it's included by cesium, if you want I can add it in the package.json?